### PR TITLE
Removed Hardware Requirement Calculator sheet

### PR DESCRIPTION
**Pull request by atultherajput**
### :nut_and_bolt: Description: What code changed, and why?

Removed Hardware Requirement Calculator sheet.

### :chains: Related Resources
https://chefio.atlassian.net/browse/CHEF-2934
### :+1: Definition of Done
https://deploy-preview-8082--chef-automate.netlify.app/automate/ha_performance_benchmarks/
https://deploy-preview-8082--chef-automate.netlify.app/automate/ha_on_premises_deployment_prerequisites/#hardware-requirements
https://deploy-preview-8082--chef-automate.netlify.app/automate/ha_aws_deployment_prerequisites/#hardware-requirements
https://deploy-preview-8082--chef-automate.netlify.app/automate/ha_aws_deploy_steps/#deployment
https://deploy-preview-8082--chef-automate.netlify.app/automate/ha_aws_managed_deploy_steps/#run-these-steps-on-bastion-host-machine
### :white_check_mark: Checklist

**All PRs** must tick these:

- [ ] I have read the [CONTRIBUTING document](https://github.com/chef/automate/blob/master/CONTRIBUTING.md).
- [ ] All commits signed-off for the [Developer Certification of Origin](https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco).

With occasional exceptions, all PRs **from Progress employees** must tick these:

- [ ] Is the code clear? *(complicated code or lots of comments--subdivide and use well-named methods, meaningful variable names, etc.)*
- [ ] Consistency checked? *(user notifications, user prompts, visual patterns, code patterns, variable names)*
- [ ] Repeated code blocks eliminated? *(adapt and reuse existing components, blocks, functions, etc.)*
- [ ] Spelling, grammar, typos checked? *(at a minimum use `make spell` in any component directory)*
- [ ] Code well-formatted? *(indents, line breaks, etc. improve rather than hinder readability)*

All PRs **from Progress employees** should tick these if appropriate:

- [ ] Tests added/updated? (all new code needs new tests)
- [ ] Docs added/updated? (all customer-facing changes)

*Please add a note next to any checkbox above if you are NOT ticking it.*

### CODE REVIEW

**Review by punitmundra — APPROVED** [review]

**Review by vivek-yadav — APPROVED** [review]

File: components/docs-chef-io/content/automate/ha_on_premises_deployment_prerequisites.md [hunk 1]
```diff
@@ -75,33 +75,22 @@ Current Automate HA integrates with the following non-Chef tools:
...
 ## Hardware Requirements
 
-{{< note >}} Use a [Hardware Calculator](/calculator/automate_ha_hardware_calculator.xlsx) to check how much hardware you will need for your use case. {{< /note >}}
-
-We have some sample values based on the performance benchmarking tests to give you an apt hardware configuration. Refer to the table below to populate things in the **Hardware Calculator** according to your requirement. The table below shows the tested **assumptions** and should not be used as exact recommendations.
+{{< note >}}
 
-You can use the below assumptions in the calculator to drive into your hardware requirement:
+- Please refer to [Performance Benchmarks](/automate/ha_performance_benchmarks) for more details on the hardware requirements.
+- Make sure the hardware requirement in not lesser than the recommended [Minimum Hardware Requirement](/automate/ha_on_premises_deployment_prerequisites/#minimum-hardware-requirement)
```
Comment (dishanktiwari2501) [inline components/docs-chef-io/content/automate/ha_on_premises_deployment_prerequisites.md:81]: Replace it with:

- Make sure the hardware requirement is not lesser than the recommended [Minimum Hardware Requirement](/automate/ha_on_premises_deployment_prerequisites/#minimum-hardware-requirement)

File: components/docs-chef-io/content/automate/ha_aws_managed_deploy_steps.md [hunk 2]
```diff
@@ -127,7 +127,7 @@ Set the above prerequisites in `~/.aws/credentials` in Bastion Host:
           - The master username value which you used while creating AWS RDS Postgresql can be used for both `managed_rds_superuser_username` and `managed_rds_dbuser_username`
           - The master password value which you used while creating AWS RDS Postgresql can be used for both `managed_rds_superuser_password` and `managed_rds_dbuser_password`
       - Set the `ami_id` value, which depends on the AWS Region and the Operating System image you want to use.
-      - Use the [Hardware Requirement Calculator sheet](/calculator/automate_ha_hardware_calculator.xlsx) to get information on which instance type you will need for your load.
+      - Please refer to the [Minimum Hardware Requirement](/automate/ha_aws_deployment_prerequisites/#hardware-requirements) to get information on which instance type you will need for your load.
```
Comment (dishanktiwari2501) [inline components/docs-chef-io/content/automate/ha_aws_managed_deploy_steps.md:130]: Replace it with: 

- Refer to the [Minimum Hardware Requirement](/automate/ha_aws_deployment_prerequisites/#hardware-requirements) to get information on which instance type you need for your load.

File: components/docs-chef-io/content/automate/ha_aws_deployment_prerequisites.md [hunk 3]
```diff
@@ -83,8 +83,8 @@ Current Automate HA integrates with the following non-Chef tools:
 
 {{< note >}}
 
--   Use a [Hardware Calculator](/calculator/automate_ha_hardware_calculator.xlsx) to check how much hardware you will need for your use case.
--   Make sure the hardware requirement in not lesser than the recommended [Minimum Hardware Requirement](/automate/ha_aws_deployment_prerequisites/#minimum-hardware-requirement)
+- Please refer to [Performance Benchmarks](/automate/ha_performance_benchmarks) for more details on the hardware requirements.
```
Comment (dishanktiwari2501) [inline components/docs-chef-io/content/automate/ha_aws_deployment_prerequisites.md:86]: Replace it with: 

- Refer to [Performance Benchmarks](/automate/ha_performance_benchmarks) for more details on the hardware requirements.

File: components/docs-chef-io/content/automate/ha_aws_deploy_steps.md [hunk 4]
```diff
@@ -117,7 +117,7 @@ Run the following steps on Bastion Host Machine:
         - Set `ssh_key_pair_name`, this is the SSH Key Pair we created as Prerequisite. This value should be just name of the AWS SSH Key Pair, not having `.pem` extention. The ssh key content should be same as content of `ssh_key_file`.
         - Set `setup_managed_services` as `false`, As these deployment steps are for Non-Managed Services AWS Deployment. Default value is `false`.
         - Set `ami_id`, this value depends on your AWS Region and the Operating System Image you want to use.
-        - Please use the [Hardware Requirement Calculator sheet](/calculator/automate_ha_hardware_calculator.xlsx) to get information for which instance type you will need for your load.
+        - Please refer to the [Minimum Hardware Requirement](/automate/ha_aws_deployment_prerequisites/#hardware-requirements) to get information for which instance type you will need for your load.
```
Comment (dishanktiwari2501) [inline components/docs-chef-io/content/automate/ha_aws_deploy_steps.md:120]: Replace it with:

- Refer to the [Minimum Hardware Requirement](/automate/ha_aws_deployment_prerequisites/#hardware-requirements) to get information on which instance type you need for your load.

File: components/docs-chef-io/content/automate/ha_on_premises_deployment_prerequisites.md [hunk 5]
```diff
@@ -75,33 +75,22 @@ Current Automate HA integrates with the following non-Chef tools:
 
 ## Hardware Requirements
 
-{{< note >}} Use a [Hardware Calculator](/calculator/automate_ha_hardware_calculator.xlsx) to check how much hardware you will need for your use case. {{< /note >}}
-
-We have some sample values based on the performance benchmarking tests to give you an apt hardware configuration. Refer to the table below to populate things in the **Hardware Calculator** according to your requirement. The table below shows the tested **assumptions** and should not be used as exact recommendations.
+{{< note >}}
 
-You can use the below assumptions in the calculator to drive into your hardware requirement:
+- Please refer to [Performance Benchmarks](/automate/ha_performance_benchmarks) for more details on the hardware requirements.
```
Comment (dishanktiwari2501) [inline components/docs-chef-io/content/automate/ha_on_premises_deployment_prerequisites.md:80]: Replace it with:

- Refer to [Performance Benchmarks](/automate/ha_performance_benchmarks) for more details on the hardware requirements.

File: components/docs-chef-io/content/automate/ha_on_premises_deployment_prerequisites.md (hunk 1 above)
Comment (dishanktiwari2501) [inline components/docs-chef-io/content/automate/ha_on_premises_deployment_prerequisites.md:81]: Replace it with:

- Make sure the hardware requirement is not lesser than the recommended [Minimum Hardware Requirement](/automate/ha_on_premises_deployment_prerequisites/#minimum-hardware-requirement)

File: components/docs-chef-io/content/automate/ha_on_premises_deployment_prerequisites.md [hunk 6]
```diff
@@ -75,33 +75,22 @@ Current Automate HA integrates with the following non-Chef tools:
...
-| Chef Automate     | 2     | 2    | 8   | 80 GB              | m5.large         |/tmp=5%  /root=20%|
-| Chef Infra Server | 2     | 2    | 8   | 80 GB              | m5.large         |/tmp=5%  /root=20%|
-| Postgresql DB     | 3     | 2    | 8   | 150 GB             | m5.large         |/tmp=5%  /root=20%|
-| Opensearch DB     | 3     | 2    | 8   | 58.9 GB            | m5.large         |/tmp=5%  /root=20%|
-| Bastion Machine   | 1     | 2    | 8   | 150 GB             | m5.large         |/tmp=5%  /root=20%|
+| Instance          | Count | vCPU | RAM | Storage Size(/hab) | AWS Machine Type | Additional Space  |
+| ----------------- | ----- | ---- | --- | ------------------ | ---------------- | ----------------- |
+| Chef Automate     | 2     | 2    | 8   | 200 GB             | m5.large         | /tmp=5% /root=20% |
+| Chef Infra Server | 2     | 2    | 8   | 200 GB             | m5.large         | /tmp=5% /root=20% |
+| Postgresql DB     | 3     | 2    | 8   | 200 GB             | m5.large         | /tmp=5% /root=20% |
```
Comment (dishanktiwari2501) [inline components/docs-chef-io/content/automate/ha_on_premises_deployment_prerequisites.md:91]: Replace Postgresql with PostgreSQL

File: components/docs-chef-io/content/automate/ha_on_premises_deployment_prerequisites.md [hunk 7]
```diff
@@ -75,33 +75,22 @@ Current Automate HA integrates with the following non-Chef tools:
...
+{{< /note >}}
 
-The machine requirements based on the above assumptions are listed below:
+### Minimum Hardware Requirement
 
-| Instance          | Count | vCPU | RAM | Storage Size(/hab) | AWS Machine Type | Additional Space |
-|-------------------|-------|------|-----|------------------- |------------------|------------------|
-| Chef Automate     | 2     | 2    | 8   | 80 GB              | m5.large         |/tmp=5%  /root=20%|
-| Chef Infra Server | 2     | 2    | 8   | 80 GB              | m5.large         |/tmp=5%  /root=20%|
-| Postgresql DB     | 3     | 2    | 8   | 150 GB             | m5.large         |/tmp=5%  /root=20%|
```
Comment (dishanktiwari2501) [inline components/docs-chef-io/content/automate/ha_on_premises_deployment_prerequisites.md:102]: Replace Postgresql with PostgreSQL

File: components/docs-chef-io/content/automate/ha_on_premises_deployment_prerequisites.md [hunk 8]
```diff
@@ -75,33 +75,22 @@ Current Automate HA integrates with the following non-Chef tools:
...
-| Opensearch DB     | 3     | 2    | 8   | 58.9 GB            | m5.large         |/tmp=5%  /root=20%|
```
Comment (dishanktiwari2501) [inline components/docs-chef-io/content/automate/ha_on_premises_deployment_prerequisites.md:103]: Replace Opensearch with OpenSearch

**Review by dishanktiwari2501 — COMMENTED** [review]
@atultherajput 
Minor Changes are Suggested!

File: components/docs-chef-io/content/automate/ha_on_premises_deployment_prerequisites.md (hunk 1 above)
Comment (atultherajput) [reply components/docs-chef-io/content/automate/ha_on_premises_deployment_prerequisites.md:81]: done

File: components/docs-chef-io/content/automate/ha_aws_deployment_prerequisites.md (hunk 3 above)
Comment (atultherajput) [reply components/docs-chef-io/content/automate/ha_aws_deployment_prerequisites.md:86]: done.

File: components/docs-chef-io/content/automate/ha_aws_deploy_steps.md (hunk 4 above)
Comment (atultherajput) [reply components/docs-chef-io/content/automate/ha_aws_deploy_steps.md:120]: done

File: components/docs-chef-io/content/automate/ha_on_premises_deployment_prerequisites.md (hunk 5 above)
Comment (atultherajput) [reply components/docs-chef-io/content/automate/ha_on_premises_deployment_prerequisites.md:80]: done

File: components/docs-chef-io/content/automate/ha_on_premises_deployment_prerequisites.md (hunk 1 above)
Comment (atultherajput) [reply components/docs-chef-io/content/automate/ha_on_premises_deployment_prerequisites.md:81]: done

File: components/docs-chef-io/content/automate/ha_on_premises_deployment_prerequisites.md (hunk 6 above)
Comment (atultherajput) [reply components/docs-chef-io/content/automate/ha_on_premises_deployment_prerequisites.md:91]: done

File: components/docs-chef-io/content/automate/ha_on_premises_deployment_prerequisites.md (hunk 7 above)
Comment (atultherajput) [reply components/docs-chef-io/content/automate/ha_on_premises_deployment_prerequisites.md:102]: done

File: components/docs-chef-io/content/automate/ha_on_premises_deployment_prerequisites.md (hunk 8 above)
Comment (atultherajput) [reply components/docs-chef-io/content/automate/ha_on_premises_deployment_prerequisites.md:103]: done

**Review by dishanktiwari2501 — APPROVED** [review]
Review Changes Made!

**Pull request merged**